### PR TITLE
Added: Ordering component for filter renderers

### PR DESCRIPTION
**Pull request by Miexil**
Refactored Date, Text & Numeric Filter renderers

### What does this PR do?

Created Ordering component for filter renderers that is located in the filter-renderers/common folder
Refactored Text, Date & Numeric filter renderers to use this new component
Added tests for this component

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated tests
- [ ] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: addon/components/hyper-table-v2/filtering-renderers/text.hbs [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
 
   {{#if @column.definition.filterable}}
-    <div class={{if @column.definition.orderable "margin-top-xx-sm"}} data-control-name={{concat "hypertable__column_filtering_for_" @column.definition.key "_filters"}}>
+    <div class={{if @column.definition.orderable "margin-top-xx-sm"}}
+         data-control-name={{concat "hypertable__column_filtering_for_" @column.definition.key "_filters"}}>
       <div class="margin-top-xxx-sm">
         <label>{{t "hypertable.column.filtering.search_term.label"}}</label>
         <OSS::InputContainer
-          @value={{this.searchQuery}} @placeholder={{t "hypertable.column.filtering.search_term.placeholder"}} />
+                             @value={{this.searchQuery}}
```
Comment (JulienVannier66) [inline addon/components/hyper-table-v2/filtering-renderers/text.hbs:14]:
```suggestion
<OSS::InputContainer @value={{this.searchQuery}}
```
Set the first args to the same line as component name, what do you think? (other in file)

**Review by JulienVannier66 — COMMENTED** [review]
Otherwise, LGTM

File: addon/components/hyper-table-v2/filtering-renderers/common/ordering.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+import TableHandler from '@upfluence/hypertable/core/handler';
+import { Column, OrderDirection } from '@upfluence/hypertable/core/interfaces';
+
+interface HyperTableV2FilteringRenderersOrderingArgs {
+  handler: TableHandler;
+  column: Column;
+  customOrderingOptions: { [key: string]: OrderDirection };
```
Comment (phndiaye) [inline addon/components/hyper-table-v2/filtering-renderers/common/ordering.ts:10]: nit:
why not call it `orderingOptions` too? that way we can tell the difference by checking if it's `this.args.orderingOptions` or `this.orderingOptions` :)

**Review by phndiaye — COMMENTED** [review]
small (nitpick) comment, thanks for this one !

**Review by JulienVannier66 — APPROVED** [review]

**Review by phndiaye — APPROVED** [review]

**Pull request merged**